src/ResultsActivity/elements/PayOtherForm/PayOtherForm.test.js

```diff
@@ -3,6 +3,11 @@ import { mount } from 'enzyme'
 import { PayOtherForm } from './PayOtherForm'
 import IEPCreated from '../../../IEPCreated'
 
+const buildLargeText = length =>
+  Array.from([...Array(length).keys()])
+    .map(_ => 'A')
+    .join('')
```

Comment:
totally meh, could you use [repeat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat)?  `length => 'A'.repeat(length)`

Reply:
Nice, didn't know you could do that!